### ISSUE TITLE
Provenance in Ordino is tracked at wrong level of abstraction

### ISSUE THREAD

**Issue by alexsb** [datavisyn/tdp_core#523]
* Environment (local or deployed): deployed

### Steps to reproduce

1. Create a combined column in Ordino
2. Provenance tracks this as two instances of remove column and one instance of add column: 
![image](https://user-images.githubusercontent.com/2039375/126644986-98c6b9f8-5e0b-496b-8bc2-a1e3d0189469.png)

### Observed behavior
* Consequence is that an immediate undo is not possible.
* One has to understand the implementation to be able to understand what happened at all. 

### Expected behavior
* A single event "Create combined column" should be created.

**Comment by thinkh** [comment]
Currently, we are just tracking the events that we recive from LineUp. But LineUp does not provide a dedicated event for creating a combined column, but only dispatches the two remove and one add events. Furthermore, we do not have a compound event that can contain multiple actions.

**Comment by alexsb** [comment]
I don't think we want a compound event, but LineUp should report this as a dedicated event.
